src/utils/log.js

```diff
@@ -32,17 +32,21 @@ const stepBanners = [
   },
   {
     name: 'Step 4',
-    description: 'Install, build and test the module, this may take a minute',
+    description: 'Initialize git',
   },
   {
     name: 'Step 5',
-    description: 'Initialize git',
+    description: 'Install, build and test the module, this may take a minute',
+  },
+  {
+    name: 'Step 6',
+    description: 'Initial Commit',
   },
 ];
 
 const fullBanner = `
- _   _     _               _____                    _       _       
-| | | |___(_)_ __   __ _  |_   _|__ _ __ ___  _ __ | | __ _| |_ ___ 
+ _   _     _               _____                    _       _
+| | | |___(_)_ __   __ _  |_   _|__ _ __ ___  _ __ | | __ _| |_ ___
```

Comment:
Did this change the look of the banner ?

Reply:
nope banner still looked good to me

Reply:
Can this change be removed ?

Reply:
do you think we need the extra whitespace ?

Reply:
Guess not. If it looks good then we can leave it.